test/integration/targets/yum_repository/aliases

```diff
@@ -1,3 +1,6 @@
 shippable/posix/group1
 destructive
 skip/aix
+# Fedora's COPR doesn't build for PowerPC so we can't test the yum_repository
+# copr functionality
+skip/power/centos
```

Comment:
Maybe we could have the tests in a `block` which is conditionalized on `ansible_architecture`, rather than skipping the arch for all tests here?